MANIFEST.in

```diff
@@ -3,10 +3,11 @@ include LICENSE
 include pyuvdata/VERSION
 include pyuvdata/GIT_INFO
```

Comment:
Do we still need this file in the manifest?

Reply:
That's a good question. I have no idea. @steven-murray ??

Reply:
Nope.